anaUltraScurve.py

```diff
@@ -326,18 +323,20 @@ def overlay_fit(VFAT, CHAN):
 
 canv.SaveAs(filename+'/Summary.png')
 
-canv = r.TCanvas('canv','canv',500*8,500*3)
-canv.Divide(8,3)
-r.gStyle.SetOptStat(0)
-for vfat in fitSums.keys():
+if options.SaveFile:
+    canv = r.TCanvas('canv','canv',500*8,500*3)
+    canv.Divide(8,3)
     r.gStyle.SetOptStat(0)
-    canv.cd(vfat+1)
-    fitSums[vfat].Draw('ap')
-    canv.Update()
+    for vfat in fitSums.keys():
```

Comment:
no reason to have .keys() here, that is what python uses by default when you iterate over a dict.